pkg/segment/structs/segsearchstructs.go

```diff
@@ -375,6 +375,26 @@ func extractSearchQueryFromMatchFilter(match *MatchFilter) *SearchQuery {
 	} else {
 		currQuery.SearchType = MatchWords
 	}
+	if len(match.MatchWords) > 0 {
```

Comment:
you can ignore the matchwords case, because matchwords case would contain many words to be matched, and then you will have to create a regex array for each of the word. So for now just ignore this case and only fill in the `Regexp` for matchphrase